src/shared/utils/color.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Convert a random string to an rgb colour
+ * @param str the string to convert
+ * @param cap an optional maximum value for r,g and b
+ * @returns rgb values as a string
+ */
+export const stringToColour = (str: string, cap = 255) => {
```

Comment:
Not sure about generating a completely random `rgb` value.
I would ideally prefer to generate something in the `lch` colour space, but support is lower I believe, but `hsl` is widely supported.
Better logic would be to hash the string to generate a hue (0-360) and stick to either fixed saturation and lightness values, or also use the hashing to generate a range within "nice" values for those (40-60% range?)

Reply:
I think the current implementation is ok for now, but will create a Jira with your suggestions. There are vague plans to unify all feature colours at some point in the future and use luminance or another property to differentiate them.